### PR TITLE
Fix Screenshots

### PR DESCRIPTION
**Pull request by mattcotton**
Fix to parse screenshot names to match suite names better. Fix for #251.

### CODE REVIEW

File: lib/xcpretty/reporters/html.rb [hunk 1]
```diff
@@ -70,7 +70,9 @@ def load_screenshots
       Dir.foreach(SCREENSHOT_DIR) do |item|
         next if item == '.' || item == '..' || File.extname(item) != '.png'
 
-        suite_name = find_test_suite(item)
+        suite = item.split(".")[0]
```
Comment (supermarin) [inline lib/xcpretty/reporters/html.rb]: This can crash if `item.split(".")` is `nil`.

Comment (mattcotton) [inline lib/xcpretty/reporters/html.rb]: Apologies, my first delve into Ruby! I'll push a fix tomorrow :)

**Comment by mattcotton** [comment]
The following applies to KIF screenshots, which I saw that was the use case for the original PR anyway:

KIF screenshots are named in a <filename>, <linenumber>.png format. `load_screenshots` was using the entire screenshot file name when finding the test suite so couldn't match.

The <filename> part of the above contains the file extension ( .swift or .m) which isn't part of the Test Suite name. Splitting on "." gets us the first part, which we can then use to match a Test Suite.

**Comment by supermarin** [comment]
Thanks @mattcotton!

**Comment by vikas-thange-twc** [comment]
is this is going to work in XCTest snapshots? Please guide on how. Thanks.

**Comment by mattcotton** [comment]
This doesn't add any additional functionality that the original PR didn't have. The original PR was based around KIF screenshots and the mechanism for loading these screenshots into the HTML report relies on the filename format that KIF provides.

If I get time, I may look in to adding support for XCTest snapshots.

**Comment by supermarin** [comment]
@mattcotton thanks! I think we should support XCTest snapshots at the first place, since it's a 1st party framework and probably most widely used

**Comment by armandzwan** [comment]
Is there any progress on XCTest screenshots?

**Pull request merged**